docroot/themes/contrib/civictheme/config/install/civictheme.settings.yml

```diff
@@ -62,6 +62,8 @@ components:
     summary_length: 160
   publication_card:
     summary_length: 160
+  site_slogan:
```

Comment:
please remove this from the scheme - the data is not stored in this value - it is stored in `system.site`

Reply:
Reverted.